kotlin/internal/jvm/associates.bzl

```diff
@@ -39,7 +56,7 @@ def _get_associates(ctx, associates):
         jars = []
         module_names = []
         for a in associates:
-            jars.append(depset(transitive = [a[JavaInfo].compile_jars, a[_KtJvmInfo].module_jars]))
+            jars.append(_collect_associates(ctx = ctx, toolchains = toolchains, associate = a))
```

Comment:
Unclear if the `a[_KtJvmInfo].module_jars` still needs to be included here. We don't hit this case in our codebase and I don't have enough historical context around why we need this here.